### PR TITLE
New term entry for Java.nextDown()

### PR DESCRIPTION
**Pull request by vaisrinivasan**
### Description

Added nextDown.md file under java/concepts/math-methods/terms/. The file
1. Defines .nextDown()
2. Provides the syntax
3. Provides an example
Closes https://github.com/Codecademy/docs/issues/907 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by vaisrinivasan** [comment]
@Dusch4593 : The PR is ready for review. Can you please let me know your comments? Thanks

File: content/java/concepts/math-methods/terms/next-down/next-down.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Syntax
+
+```pseudo
+Math.nextDown(x)
+```
+
+- The argument `x` can be of type `double` or `float`.
+- The return type is of `double` or `float`.
+- This method is semantically equivalent to `nextAfter(d, Double.NEGATIVE_INFINITY);` or `nextAfter(f, Float.NEGATIVE_INFINITY);` however, a nextDown implementation may run faster than its equivalent nextAfter call.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/next-down/next-down.md:30]:
```suggestion
- Though it is the opposite of [`.nextAfter()`](https://www.codecademy.com/resources/docs/java/math-methods/nextAfter), the `.nextDown()` method is semantically equivalent to `nextAfter(d, Double.NEGATIVE_INFINITY);` or `nextAfter(f, Float.NEGATIVE_INFINITY);`.

> **Note:** A `.nextDown()` implementation may run faster than its equivalent `.nextAfter()` call.
```

File: content/java/concepts/math-methods/terms/next-down/next-down.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Special Cases:
+
+- If the argument is NaN, the result is NaN.
+- If the argument is negative infinity, the result is negative infinity.
+- If the argument is zero, the result is `-Double.MIN_VALUE` or `-Float.MIN_VALUE`.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/next-down/next-down.md:36]:
```suggestion
Special cases include the following:

- If `x` is `NaN`, the result is `NaN`.
- If `x` is negative infinity, the result is negative infinity.
- If `x` is zero, the result is `-Double.MIN_VALUE` or `-Float.MIN_VALUE`.
```

File: content/java/concepts/math-methods/terms/next-down/next-down.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## Example
+
+The following example demonstrates using `Math.nextDown(x)`:
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/next-down/next-down.md:40]:
```suggestion
The following is an example of the `.nextDown()` method:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@vaisrinivasan Finished with my review and left you some comments/suggestions. 😄

**Comment by vaisrinivasan** [comment]
@Dusch4593 : Have committed your suggestions. 👍 Requesting for further steps to proceed

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
One more request, @vaisrinivasan. Can we change the names of the folder and file for this entry (from `next-down` and `next-down.md` to `nextDown` and `nextDown.md`)?

**Review by Dusch4593 — APPROVED** [review]
@vaisrinivasan Thanks! Looks good for the next review. 👍🏻

**Review by SSwiniarski — APPROVED** [review]
@vaisrinivasan, @Dusch4593 Looks good to me.

**Comment by Dusch4593** [comment]
cc: @SSwiniarski @vaisrinivasan

Prior to merge, I made a small change to indentation in the \```java code block in 2e15c8bea8317532303ccd1ab90f94b2404e67fb (2 spaces instead of 4 spaces).

**Pull request merged**